sailfish-browser.desktop

```diff
@@ -6,6 +6,10 @@ X-MeeGo-Translation-Catalog=sailfish-browser
 Icon=icon-launcher-browser
 Exec=/usr/bin/sailjail -p sailfish-browser.desktop /usr/bin/sailfish-browser %U
 Comment=Sailfish UI application
+MimeType=text/html;x-scheme-handler/http;x-scheme-handler/https;
```

Comment:
Why have you dropped `text/xml;application/xhtml+xml;application/xml`?

Reply:
I do not know. I thought I'd just copied as if from the removed .desktop file.

Reply:
Got it.